app/controllers/v1/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# API for Profiles
+module V1
+  class ProfilesController < ::ProfilesController
+  end
```

Comment:
This should be superclassed by a `V1::ApplicationController`, and we should copy everything from existing controllers into these V1 controllers and delete the old controllers. I guess there may still be some common logic between the versions, but any common logic that is slightly different for one version over the other will be difficult to maintain without breaking one of the APIs.